src/Sentry.Unity.Editor/SentryTestWindow.cs

```diff
@@ -14,7 +14,7 @@ public static SentryTestWindow Open()
         public void Dispose()
         {
             Close(); // calls 'OnLostFocus' implicitly
-            File.Delete(SentryOptionsAssetPath);
+            File.Delete(UnitySentryOptions.GetConfigPath());

```

Comment:
You kinda broke tests :) You must use `SentryTestWindow.SentryOptionsAssetName` path.